### PR TITLE
[JAVA-896] Removed @ResponseBody by replacing @Controller with @RestC…

### PR DESCRIPTION
**Pull request by aurasphere**
…ontroller.

### CODE REVIEW

File: src/main/java/com/baeldung/web/controller/OldRegistrationController.java [hunk 1]
```diff
@@ -27,18 +27,17 @@
...
 import org.springframework.web.bind.annotation.ModelAttribute;
 import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestMethod;
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.servlet.ModelAndView;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PostMapping;
 
-@Controller
+@RestController
```
Comment (lor6) [inline src/main/java/com/baeldung/web/controller/OldRegistrationController.java]: is this change needed here, since the mapping in this controller don't have a ResponseBody annotation, and return views instead?

File: src/main/java/com/baeldung/web/controller/UserController.java [hunk 2]
```diff
@@ -2,14 +2,15 @@
...
-import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.baeldung.security.ActiveUserStore;
+import com.baeldung.service.IUserService;
 
-@Controller
+@RestController
```
Comment (lor6) [inline src/main/java/com/baeldung/web/controller/UserController.java]: is this change needed here, since the mapping in this controller don't have a ResponseBody annotation, and return views instead?

Comment (aurasphere) [reply src/main/java/com/baeldung/web/controller/UserController.java]: Technically no, I've just changed it for the sake of consistency. Let me know if I should revert this  change. Thanks

File: src/main/java/com/baeldung/web/controller/OldRegistrationController.java (hunk 1 above)
Comment (aurasphere) [reply src/main/java/com/baeldung/web/controller/OldRegistrationController.java]: Technically no, I've just changed it for the sake of consistency. Let me know if I should revert this  change. Thanks

Comment (jzheaux) [reply src/main/java/com/baeldung/web/controller/OldRegistrationController.java]: Let's leave it as `@Controller`. `@RestController` is for controllers that return data.
